README.md

```diff
@@ -1,31 +1,27 @@
-<!--a href="https://microclimate-dev2ops.github.io/">
-    <img src="mc-banner.png" alt="Microclimate Banner" height="75px" />
-</a-->
-
-# Codewind for VS Code
-
 [![Marketplace](https://img.shields.io/vscode-marketplace/v/IBM.codewind.svg?label=marketplace&logo=visual-studio-code)](https://marketplace.visualstudio.com/items?itemName=IBM.codewind)
 [![License](https://img.shields.io/badge/License-EPL%202.0-red.svg?label=license&logo=eclipse)](https://www.eclipse.org/legal/epl-2.0/)
 [![Slack](https://img.shields.io/badge/ibm--cloud--tech-blue.svg?logo=slack&label=slack)](https://slack-invite-ibm-cloud-tech.mybluemix.net/)
 
-- **[Documentation](https://microclimate.dev/codewindtechpreview)**
-- **[Changelog](https://github.com/microclimate-dev2ops/codewind-vscode/blob/master/CHANGELOG.md)**
-
+# Codewind for VS Code
 Create and develop cloud-native, containerized web applications from VS Code.
 
-## Getting started
+## Installing Codewind
+Prerequisites
+- Install [VS Code](https://code.visualstudio.com/download).
+- Install Docker.
+- If you use Windows, you also need to install Docker Compose.
```

Comment:
windows or linux

Reply:
technically "if you are not on macOS" you also need to install docker-compose

Reply:
Sorry I'm wrong, it's actually *only* linux that doesn't include docker-compose ootb